packages/manager/apps/dedicated/client/app/account/identity-documents/components/identity-documents-corporation/template.html

```diff
@@ -0,0 +1,96 @@
+<p
+    class="oui-heading_3 mb-1 mt-3"
+    data-translate="user_account_identity_documents_corporation_requirements"
+></p>
+<p
+    data-translate="user_account_identity_documents_corporation_requirements_intro"
+></p>
+<p>
+    <span
+        class="font-weight-bold"
+        data-translate="user_account_identity_documents_corporation_requirements_identity"
+    ></span>
+    <span
+        data-translate="user_account_identity_documents_corporation_requirements_identity_list"
+    ></span>
+</p>
+<p
+    data-translate="user_account_identity_documents_corporation_requirements_acceptance_criteria"
+></p>
+<ul>
+    <li>
+        <span data-translate="user_account_identity_documents_corporation_requirements_identity_acceptance_criteria_1"></span>
+    </li>
+    <li>
+        <span data-translate="user_account_identity_documents_corporation_requirements_identity_acceptance_criteria_2"></span>
+    </li>
+    <li>
+        <span data-translate="user_account_identity_documents_corporation_requirements_identity_acceptance_criteria_3"></span>
+    </li>
+    <li>
+        <span data-translate="user_account_identity_documents_corporation_requirements_identity_acceptance_criteria_4"></span>
+    </li>
```

Comment:
**nitpick:** Content can be directly included in the `li` and `span` can be avoided.
```suggestion
    <li data-translate="user_account_identity_documents_corporation_requirements_identity_acceptance_criteria_1">
    </li>
    <li data-translate="user_account_identity_documents_corporation_requirements_identity_acceptance_criteria_2">
    </li>
    <li data-translate="user_account_identity_documents_corporation_requirements_identity_acceptance_criteria_3">
    </li>
    <li data-translate="user_account_identity_documents_corporation_requirements_identity_acceptance_criteria_4">
    </li>
```